src/termData.ts

```diff
@@ -67,3 +72,5 @@ function getDefaultTerm() {
 }
 
 export { termData, defaultTerm, getDefaultTerm };
+
+//https://tenor.com/view/leonardo-dicaprio-rick-dalton-point-finger-pointing-hand-right-there-gif-20349799
```

Comment:
what lmao

Reply:
was pasting that somewhere else and accidentally put it in the project I guess 🥴